### PR TITLE
You can now point while handcuffed

### PR DESCRIPTION
**Pull request by azzzertyy**
# Document the changes in your pull request
as title

# Why is this good for the game?
small quality of life change which makes sense because cuffs dont magically prevent you from pressing E and typing "points at the airlock"
# Changelog
:cl:  
tweak: You can now point while handcuffed
/:cl:

### CODE REVIEW

**Comment by SomeguyManperson** [comment]
mime mains when they can't spam point

**Comment by azzzertyy** [comment]
> mime mains when they can't spam point

idk how a handcuff can magically cuff your fingers together man

**Review by SapphicOverload — CHANGES_REQUESTED** [review]
"Why is this good for the game" section should contain an actual reason for why you think it's good for the game, also does this work with straight jackets too?

**Comment by azzzertyy** [comment]
> "Why is this good for the game" section should contain an actual reason for why you think it's good for the game, also does this work with straight jackets too?

probably for straightjackets, i can remove if its silly

do i really have to make a write up for why this is good for the game?
i'd much prefer people had their own opinions rather than i tried to convince them

**Comment by iamthesilly** [comment]
> probably do i really have to make a write up for why this is good for the game? i'd much prefer people had their own opinions rather than i tried to convince them

It can be like one sentence or something, just has to be any reason at all.

**Review by SapphicOverload — COMMENTED** [review]
Good enough

**Comment by Therandomhoboo** [comment]
No

**Comment by azzzertyy** [comment]
> Good enough

straightjackets now prevent pointing again

**Comment by cuackles** [comment]
Make it fuck up a bit the direction of the pointing cus hands tied to close

**Comment by azzzertyy** [comment]
> Make it fuck up a bit the direction of the pointing cus hands tied to close

way too much effort sadly

**Comment by cuackles** [comment]
> > Make it fuck up a bit the direction of the pointing cus hands tied to close
> 
> way too much effort sadly

Time to abuse my director powers to close this then  😔

**Comment by Hopekz** [comment]
> > probably do i really have to make a write up for why this is good for the game? i'd much prefer people had their own opinions rather than i tried to convince them
> 
> It can be like one sentence or something, just has to be any reason at all.

You're really sounding like a secretary vs a maintainer

**Comment by SomeguyManperson** [comment]
Because our playerbase clearly needs yet more ways to draw attention to themselves while handcuffed, as if scream spamming isnt enough already (and at least you can muzzle them to stop that)

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -367,7 +367,10 @@
 
 //same as above
 /mob/living/pointed(atom/A as mob|obj|turf in view())
-	if(incapacitated())
+	var/obj/item/clothing/suit/straightjacket = src.get_item_by_slot(ITEM_SLOT_OCLOTHING)
```
Comment (Moltijoe) [inline code/modules/mob/living/living.dm:370]: gotta give the variable an actual name, currently it's a /obj/item/clothing/suit variable that's named straightjacket
meaning literally any /suit item will prevent pointing

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Comment by azzzertyy** [comment]
Something came up. Cant finish working on this right now.

**Pull request closed without merging**